toolkit/tools/imagecustomizer/main.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/imagecustomizerlib"
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/exe"
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/logger"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+var (
+	app = kingpin.New("imagecustomizer", "Customizes a pre-built CBL-Mariner image")
+
+	buildDir          = app.Flag("build-dir", "Directory to run build out of.").Required().String()
```

Comment:
We haven't been super consistent in our older tools, but kingpin has things like `ExistingFile()` which can simplify stuff later in the tool, depending on how you want the args to be used.

Reply:
The current code actually creates the `buildDir` directory if it doesn't exist.